README.md

```diff
@@ -30,7 +30,7 @@ The YugabyteDB connector can also be used as a library without Kafka or Kafka Co
   4. Copy the debezium-connector-yugabytedb2 jar to the custom-connector directory
       
       ```sh
-      cp debezium-connector-yugabytedb2/target/debezium-connector-yugabytedb-1.3.2-BETA.jar ~/custom-connector/
+      cp target/debezium-connector-yugabytedb-1.7.0.8-SNAPSHOT.jar ~/custom-connector/
```

Comment:
Can you replace with a wildcard? For example

    cp target/debezium-connector-yugabytedb-*.jar ~/custom-connector